automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala

```diff
@@ -70,6 +70,11 @@ class BillingSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Cl
               val workspaceName = "BillingSpec_makeWorkspace_" + randomUuid
               register cleanUp api.workspaces.delete(billingProjectName, workspaceName)
               val detailPage = listPage.open.createWorkspace(billingProjectName, workspaceName)
+
+              detailPage.validateWorkspace shouldEqual true
+
+              listPage.open
+              listPage.hasWorkspace(billingProjectName, workspaceName) shouldBe true
```

Comment:
Checking workspace successfully created here, removed redundant test `"with a billing project, should be able to create a workspace"` in `WorkspaceOwnerSpec`